src/promo/ThinDonationBanner.jsx

```diff
@@ -3,20 +3,21 @@ import ArrowRightIcon from '@material-ui/icons/KeyboardArrowRight'
 import PropTypes from 'prop-types'
 import React from 'react'
 import withStyles from '@material-ui/core/styles/withStyles'
+import corePalette from '../styles/palettes/core'
 
 const styles = theme => ({
   button: {
     borderRadius: 0,
     justifyContent: 'left',
-    backgroundColor: theme.palette.core && theme.palette.core.main,
+    backgroundColor: corePalette[600],
 
     // We need to mark the colour with `!important` to avoid global hyperlink
     // styles overriding this value.
-    color: [theme.palette.primary.contrastText, '!important'],
+    color: ['#fff', '!important'],
```

Comment:
Could/should this use `neutralPalette[0]` here?

Reply:
Ahhh, must've messed up a rebase. Yes.

Reply:
@RohanM Done. Force pushed